app/api/api_v1/endpoints/user.py

```diff
@@ -35,7 +35,7 @@ def register(
     if settings.USING_GATEKEEPER:
         try:
             response = requests.post(
-                url=str(settings.GATEKEEPER_BASE_URL) + "/api/register/",
+                url=str(settings.GATEKEEPER_BASE_URL) + "api/register/",
```

Comment:
I think that it is better to `strip()` any trailing slash from GATEKEEPER_BASE_URL and then a the url path like. This solutionn is more robust 

```
url=str(settings.GATEKEEPER_BASE_URL).strip('/') + "/api/register/",
```